File: SoundDemo/app/src/main/java/com/example/sounddemo/MainActivity.java
package com.example.sounddemo;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.media.MediaPlayer;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    private MediaPlayer mPlayer;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        mPlayer = MediaPlayer.create(this, R.raw.tasmania);
    }

    public void playBtt(View view) {
        mPlayer.start();
    }

    public void pauseBtt(View view) {
        mPlayer.pause();
    }

}

File: Ceep/app/src/main/java/org/caio/ceep/MainActivity.kt
package org.caio.ceep

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.recyclerview.widget.RecyclerView
import androidx.recyclerview.widget.StaggeredGridLayoutManager
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    private var recyclerVew : RecyclerView? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        components()
    }

    private fun components() {
        recyclerVew = listNotesRecycler
        recyclerVew?.let {
            recyclerVew!!.adapter = NoteListAdapter(notes(), this)

            val layoutManager = StaggeredGridLayoutManager(2,
                StaggeredGridLayoutManager.VERTICAL)
            recyclerVew!!.layoutManager = layoutManager
        }
    }

    private fun notes(): List<Note> {
        return listOf(Note("Leitura", "Livro de Kotlin com Android"),
        Note("Pesquisa", "The clean code bible\n scasdas"),
        Note("Transire", "Instituto de Tecnologia e BioTecnologia do Amazonas"))
    }
}